docs/search-views/fulltext-search/getting-started-search.md

```diff
@@ -6,35 +6,39 @@ title: Getting Started with Fulltext Search
+- The `addresses` collection created during the first two steps of the [Quickstart](https://www.macrometa.com/docs/quickstart).
 
-## Create a Search View
+## 1. Create a Search View
 
-Refer to the [Fulltext Search Tasks](tasks/index.md) section for an explanation of how search views function.
+You are going to create a search view on the document collection that you created in the [Quickstart](https://www.macrometa.com/docs/quickstart). For an explanation of how search views function, refer to the [Fulltext Search Tasks](tasks/index.md) section.
 
 1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
-2. Click **Search Views**.
+2. Click **Data > Search Views**.
 3. Click **New View**.
-4. In the **Name** field, enter `addresses_view`.
-5. Click the **plus** under Mapping Definition.
-6. Select the `addresses` collection from the COLLECTION list.
+4. In the **Search View Name** field, enter `addresses_view`.
+5. In **View Type**, select **Fulltext Search**.
+6. In the **Collection** list, select **addresses**.
 7. Enter `lastName` in `field` and select **text_en** in the ANALYZER list.
-8. Perform the previous three steps again for `firstName` and `email`.
-9. Click **Create**.
+8. Click **Add Collection** twice.
+9. Perform steps 6 and 7 again for the `firstName` and `email` fields in the `addresses` collection.
+10. Click **Create**.
 
-Your three address collection key values are indexed and ready to query. Macrometa distributes this view index to every location in the global fabrics. If you're curious about the locations, click **Dashboard** to see the default [GeoFabric](geofabrics/index.md) locations.
+Your three address collection key values are indexed in the search view and ready to query. Macrometa distributes this search view to every location in the global fabrics. If you're curious about the locations, click **Dashboard** to see the default [GeoFabric](geofabrics/index.md) locations.
 
-## Query the View
+![New Search View](/img/search/getting-started-new-fulltext.png)
 
-To query the search view, we can create a query in Macrometa to run it on the fly or save it as a [query worker](../../queryworkers/index.md).
+## 2. Query the Search View
 
-1. On the side menu, click **Query Workers**.
-1. The code block below is a search query that will allow you to pass in a query parameter to search you index. Copy and paste it in the code editor on line 1.
+To query the search view, you can create a query in Macrometa to run it as needed or save it as a [query worker](../../queryworkers/index.md).
+
+1. On the side menu, click **Compute > Query Workers**.
+2. If the code editor is not empty, then click **New Query**.
+3. The code block below is a search query that will allow you to pass in a query parameter to search the search view. Copy and paste it in the code editor on line 1.
 
   ```sql
     FOR contact IN addresses_view
```

Comment:
@amolth-macrometa, @maheshw2, @koshy-thomas , @Luba89, @elof - Any idea why this doesn't work? See my description for more details.

Reply:
@diana-macrometa it works for me using nemo smallcase check for trailing spaces maybe? (since text_en analyzers since  text analyzer bydefault lower cases it (https://www.arangodb.com/docs/stable/analyzers.html)
)

Reply:
No trailing spaces, and it still isn't returning anything. Maybe we could pair today or tomorrow and you see if you can spot what I am doing wrong?